phpunit.xml

```diff
@@ -37,10 +37,7 @@
         <env name="SESSION_DRIVER" value="array"/>
         <env name="QUEUE_CONNECTION" value="sync"/>
         <env name="MAIL_DRIVER" value="array"/>
-        <env name="DB_CONNECTION" value="mysql_testing"/>
-        <env name="DB_TEST_DATABASE" value="restarters_db"/>
-        <env name="DB_TEST_USERNAME" value="restarters"/>
-        <env name="DB_TEST_PASSWORD" value="s3cr3t"/>
+        <env name="DB_CONNECTION" value="mysql"/>
```

Comment:
I had this as locally I have a dev DB (to test things while coding them, or just testing another PR) that has close to prod data in it, and a separate testing DB.  

If I run tests locally now this would truncate tables in my dev DB I think.  How do you have this set up on your box?

Reply:
Ah, ok - I didn't realise it was actively used.  

When I run the tests it does indeed wipe my database, which I live with. So that use of a separate environment doesn't seem to be working for me, which I don't understand.  But I'll reinstate that stuff and then regenerate my Docker environment to check it still works.